sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/datasources/samples/SamplesDbConnector.groovy

```diff
@@ -107,23 +106,25 @@ class SamplesDbConnector implements CountSamplesDataSource, DownloadSamplesDataS
       Connection connection = connectionProvider.connect()
       String latestChangeQuery = "select MAX(UNIX_TIMESTAMP(arrival_time)) from samples_locations where sample_id LIKE ?"
       String sqlRegex = "${projectCode}%"
-      Timestamp latest = new Timestamp(0)
+      Instant latest = Instant.MIN
```

Comment:
What does Instant.Min do? create a instant of a minute to initialize the value?

Reply:
It's like Integer.MIN_VALUE and guarantees that projects without a recorded change show up last when sorted by that time.

Reply:
very neat, thanks for that explanation!